### DISCUSSION TITLE
Tally Arbiter over Internet via DNS and port forwarding

### DISCUSSION THREAD

**Discussion by DAMIGUI** [josephdadams/TallyArbiter#384] [Q&A] [answered]
Hello Guys !

We would like to transmit tally over internet via a tally arbiter behind a domaine name and a port forwarding over 443.
(The web interface works correctly from the outside)

Unfortunately the remote server could not connect to the cloud server behind a domaine name.
Is there a particular configuration to set up?

Thanks for your help.
Best Regards

**Comment by hrueger** [comment]
Are you using v3?
It is a known issue that cloud connection is not working there (yet)...

**Reply by DAMIGUI** [reply]
Hello,
Thanks for your reply.

We are using v2.0.16.
We tested the cloud connection with the public IP and the default port (4455) and it works fine.

Unfortunately using a domain name and port forwarding (4455 to 443) it no longer works.
Do you know more about this ?

Have a nice day !

**Comment by DAMIGUI** [comment, marked as answer]
Hello,
We finally discovered that Tally arbiter does not resolve a DNS. We are obligated to use IP addresses.
do you know how we could solve this?

Bye

**Reply by JTF4** [reply]
Have you tried changing the external address in the config.json file to match the URL you want to access TallyArbiter from?

**Reply by DAMIGUI** [reply]
You are right ! You must fill in the DNS in the external field.

Thank you very much
